governance/tsc/README.md

```diff
@@ -6,7 +6,7 @@ The TSC is responsible for technical oversight of the Servo Project. Check the [
 
 The TSC meets in public, and all minutes are published.
 
-* 2025: [24 Feb](tsc-2025-02-24.md), [31 Mar](tsc-2025-03-31.md)
+* 2025: [24 Feb](tsc-2025-02-24.md), [31 Mar](tsc-2025-03-31.md), [28 Apr](tsc-2025-04-28.md)
```

Comment:
Looks like the file has not been added to git.

Reply:
:facepalm: sorry. Now it's there.